.gitattributes

```diff
@@ -1,2 +1 @@
-# Auto detect text files and perform LF normalization
-* text=auto
```

Comment:
Note that `* text=auto` could be removed because we have included the pre-commit hook `mixed-line-endings` which handles the line endings